typings/index.d.ts

```diff
@@ -570,7 +570,7 @@ declare module 'discord.js' {
           format: 'default' | AllowedImageFormat,
           size: number,
         ) => string;
-        Banner: (guildID: Snowflake | number, hash: string, format: AllowedImageFormat, size: number) => string;
+        Banner: (id: Snowflake | number, hash: string, format: AllowedImageFormat, size: number) => string;
```

Comment:
This needs to be changed to 
```js
Banner: (id: Snowflake | number, hash: string, format: AllowedImageFormat, size: number, dynamic: boolean) => string;
```

Reply:
I was wondering if so and looked at the other functions, dynamic does not appear. Isn't it because dynamic is declared here ? 
https://github.com/discordjs/discord.js/blob/e8134d558d4ad5da4a37b0b120be14a15bf70b61/typings/index.d.ts#L3512

Reply:
Hmm I guess it doesn't need to added there. just from the actual function it needed to be there. But I guess not lol 👍

Reply:
`dynamic` missing is probably an oversight, I can't think of any reason it shouldn't be there.

Reply:
That was my first thought, I've added the 3 missing `dynamic`.